packages/@lwc/engine-core/src/framework/base-bridge-element.ts

```diff
@@ -104,15 +108,36 @@ function createAttributeChangedCallback(
     };
 }
 
+function createStandardAccessorDescriptor(propName: string) {
+    let prop: any;
+    return {
+        get() {
+            logWarn(
+                `The property "${propName}" is not publicly accessible. Add the @api annotation to the property declaration or getter/setter in the component to make it accessible.`
+            );
+            return prop;
+        },
+        set(value: any) {
+            logWarn(
+                `The property "${propName}" is not publicly accessible. Add the @api annotation to the property declaration or getter/setter in the component to make it accessible.`
+            );
+            prop = value;
+        },
+        enumerable: true,
+        configurable: true,
+    };
+}
+
 export interface HTMLElementConstructor {
     prototype: HTMLElement;
     new (): HTMLElement;
 }
 
 export function HTMLBridgeElementFactory(
     SuperClass: HTMLElementConstructor,
-    props: string[],
-    methods: string[]
+    publicProperties: string[],
+    methods: string[],
+    proto: LightningElement
```

Comment:
Much better! I really like that we're only adding the `proto` to the arguments here.

Reply:
There is a TypeScript-related build failure:
```
[!] (plugin typescript) RollupError: @rollup/plugin-typescript TS2345: Argument of type 'null' is not assignable to parameter of type 'LightningElement'.
src/framework/base-bridge-element.ts (216:5)

216     null
```

I think you need to add `| null` to the type so that it accepts null.

```suggestion
    proto: LightningElement | null
```

Reply:
@nolanlawson what is the name of the method you want me to use?

Reply:
@AllanOricil I think you may be responding to the wrong thread. 🙂 As for the method name, I mentioned it [here](https://github.com/salesforce/lwc/pull/3713#discussion_r1332188837).